scriptworker/cot/verify.py

```diff
@@ -608,24 +606,18 @@ def _sort_dependencies_by_name_then_task_id(dependencies):
 
     """
     mandatory_artifact_tasks = []
```

Comment:
I'd consider removing the `mandatory_` here, since that is the only thing now (but also understand if you'd rather do that separately).